### PR TITLE
1127 - Appeal details page

### PR DESCRIPTION
**Pull request by topickering**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-1127

## Description of change

Adds in appeal details on the appeal-details page.  NOTE - at present does not include related documents.

Includes a bugfix for title of appeal-details pages.

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- X My commit history in this PR is linear
- X New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/Planning-Inspectorate/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/common/src/lib/format-address.js [hunk 1]
```diff
@@ -22,6 +22,25 @@ const formatAddress = (appealCaseData) => {
 	return addressComponents.filter(Boolean).join(', ');
 };
 
+/**
+ * @param {AppealCaseWithAppellant | AppealSubmission} appealCaseData
+ * @returns {string}
+ */
+const formatAddressWithBreaks = (appealCaseData) => {
```
Comment (Si3rr4wow) [inline packages/common/src/lib/format-address.js:29]: You could have made the joiner an optional argument of the existing `formatAddress` function then just pass in `'<br>'` when you need it ;)

File: packages/common/src/lib/format-appeal-details.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ */
+exports.formatVisibility = (caseData) => {
+	const visibility = caseData.appellantSiteAccess ? 'Yes' : 'No';
+
+	if (caseData.appellantSiteAccessDetails) {
+		return `${visibility}<br>${caseData.appellantSiteAccessDetails}`;
+	}
+
+	return visibility;
+};
```
Comment (Si3rr4wow) [inline packages/common/src/lib/format-appeal-details.js:25]: It's possible to do this kind of thing without branching but this is quite terse so feel free to ignore

```suggestion
exports.formatVisibility = (caseData) => {
	const visibility = caseData.appellantSiteAccess ? 'Yes' : 'No';

	return visibility + (caseData.appellantSiteAccessDetails && `<br>${caseData.appellantSiteAccessDetails}`);
};
```

Screenshot 2024-03-08 at 14 17 29

File: packages/common/src/lib/format-address.js (hunk 1 above)
Comment (topickering) [reply packages/common/src/lib/format-address.js:32]: Changed!

File: packages/common/src/lib/format-appeal-details.js (hunk 2 above)
Comment (topickering) [reply packages/common/src/lib/format-appeal-details.js]: Ah nice, have refactored to use that here, in the formatSafety and in the formatAgent functions

**Review by Si3rr4wow — APPROVED** [review]
El Jeetum

**Pull request merged**